_data/canonical-tags/architecture.yaml

```diff
@@ -0,0 +1,3 @@
+id: architecture
+name: Architecture
+description: Related to the inner components of the Kubernetes system.
```

Comment:
> the Kubernetes system

Can we replace this with just "Kubernetes"? See `operation.yaml` for the same question. So:

> description: The inner components of Kubernetes.